src/SFA.DAS.PR.Database/Tables/Requests.sql

```diff
@@ -0,0 +1,39 @@
+﻿CREATE TABLE [dbo].[Requests]
+(
+    [Id] UNIQUEIDENTIFIER NOT NULL DEFAULT NEWID(),
+    [RequestType] VARCHAR(10) NOT NULL,
+    [Ukprn] BIGINT NOT NULL,
+    [ProviderUserFullName] NVARCHAR(255) NOT NULL,
```

Comment:
I removed ProviderUserFullName as it was not needed and replaced EmployerUser by ActionedBy (which can be either email of UUID)
![image](https://github.com/SkillsFundingAgency/das-pr-api/assets/10530268/ab65ff12-2728-4931-8dd9-98496a5e5dc7)